docs/installation/config.rst

```diff
@@ -1,109 +1,99 @@
-.. _installation_environment_config:
+.. _installation_env_config:
 
 ===================================
 Environment configuration reference
 ===================================
 
-The Objects and Objecttypes APIs can be run both as a Docker container or a VPS / dedicated server. It relies on other services, such as
-database and cache backends, which can be configured through environment
-variables.
 
-Available environment variables
-===============================
-
-Required settings
------------------
-
-* ``DJANGO_SETTINGS_MODULE``: which environment settings to use. Available options:
-
-  - ``objects.conf.docker`` or ``objecttypes.conf.docker``
-  - ``objects.conf.dev`` or ``objecttypes.conf.dev``
-  - ``objects.conf.ci`` or ``objecttypes.conf.ci``
```

Comment:
This is now removed from the docs, because it is not explicitly set using `config(...)`. Is this issue?

Reply:
I don't think so